cfme/automate/explorer/method.py

```diff
@@ -484,6 +507,15 @@ def create(
                 'playbook_input_parameters': playbook_input_parameters
             })
             validate = False
+
+        if location in ['Ansible Tower Workflow Template', 'Ansible Tower Job Template']:
```

Comment:
if self.location in self.LOCATIONS:

Reply:
I think `location` is fine there, no need for `self` as `location` is an argument in create.

Reply:
Yup that was a typo :)